### PR TITLE
Adds Learn to Code Course and Levels

### PR DESCRIPTION
**Pull request by kplawver**
* Adds Learn to Code from docs to the course list.
* Adds 4 levels with tiny little descriptions.

### CODE REVIEW

File: app/models/course.rb [hunk 1]
```diff
@@ -575,6 +575,42 @@ class Course < ActiveHash::Base
...
+          title: "Some Programming Experience, Nothing Formal",
+          level_description: [
+            'Maybe you\'re a front-end developer and written a little JavaScript, or you\'ve written a WordPress plugin.'
+          ]
+        }, {
+          level: 4,
+          color: 'orange',
+          title: 'I\'m a Professional Programmer',
+          level_description: [
+            'You should probably be volunteering instead of attending, but that\'s alright. You can come!'
```
Comment (lilliealbert) [inline app/models/course.rb]: looooool

**Comment by lilliealbert** [comment]
@kplawver Looks good! The one thing I'm wondering about is whether it should be called "Learn to Code, featuring Ruby" (or some variant), since the description mentions it's a Ruby course?

The main place that I think this matters is on the course drop down, where just "Learn to Code" looks very generic. Which could be good? IDK. But we should probably make it either fully generic or make it say Ruby in the title.

screen shot 2016-11-06 at 9 36 44 am

Whaddya think?

**Comment by kplawver** [comment]
I think adding "with Ruby" to the end makes it clearer and is more like the
Go intro class. Will update in a few.

**Comment by kplawver** [comment]
@lilliealbert Okey-dokey, all set (I think).

**Comment by tjgrathwell** [comment]
Okay, it's deployed :rocket: !

**Comment by kplawver** [comment]
:metal:  Thanks!  And thanks for fixing the course and description!

**Pull request merged**